app/components/sbom/component-details/vulnerabilities/index.scss

```diff
@@ -0,0 +1,35 @@
+.vulnerability-list-header,
+.vulnerability-detail-wrapper {
+  display: grid;
+  grid-template-columns: 3fr 2fr 1fr;
+}
+
+.vulnerability-list-header {
+  background-color: var(
+    --sbom-component-details-vulnerabilities-list-header-background-color
+  );
+  border: 1px solid
+    var(--sbom-component-details-vulnerabilities-list-header-border-color-dark);
+
+  .vulnerability-list-header-item {
+    padding: 0.75em 1.2143em;
+  }
+}
+
+.vulnerability-list-item {
+  flex-direction: column;
+  border-bottom: 1px solid
```

Comment:
Not a blocker but we can just have a border declaration and a `border-bottom` to none since they share similar properties.

Reply:
done